fvtest/compilertriltest/ShiftAndRotateTest.cpp

```diff
@@ -1017,14 +1017,27 @@ std::vector<T> test_masks();
 
 static uint64_t mask64Values[] =
     {
-    0x0000000000000000,
-    0xffffffffffffffff,
+    0x0,
+    0x1,
+    0x2,
+    0x3E,
```

Comment:
Just a minor nit: It would probably be best to format these values to be consistent with the other values in this list (lowercase hex digits with left-padding zeroes).

Reply:
Fixed!